test/integration/podman_test.go

```diff
@@ -28,15 +28,24 @@ var _ = Describe("podman preset", Serial, Ordered, Label("podman-preset"), func(
 	Describe("basic use", Serial, Ordered, func() {
 
 		It("write to config", func() {
-			Expect(RunCRCExpectSuccess("config", "set", "preset", "podman")).To(ContainSubstring("please run 'crc setup' before 'crc start'"))
+			Expect(RunCRCExpectSuccess("config", "set", "preset", "microshift")).To(ContainSubstring("please run 'crc setup' before 'crc start'"))
 		})
 
 		It("setup CRC", func() {
-			Expect(RunCRCExpectSuccess("setup")).To(ContainSubstring("Your system is correctly setup for using CRC"))
+			if bundlePath == "" {
+				Expect(RunCRCExpectSuccess("setup")).To(ContainSubstring("Your system is correctly setup for using CRC"))
+			} else {
+				Expect(RunCRCExpectSuccess("setup", "-b", bundlePath)).To(ContainSubstring("Your system is correctly setup for using CRC"))
```

Comment:
You don't need to provide the bundlePath because this is link to OCP and not microshift one. so better to just `setup` and the `start` with pull secret.

Reply:
Changed